src/components/Modals/ReplaceCourseModal.vue

```diff
@@ -0,0 +1,294 @@
+<template>
+  <teleport-modal
+    title="Replace Course"
+    content-class="content-course"
+    :leftButtonText="leftButtonText"
+    :rightButtonText="rightButtonText"
+    :rightButtonIsDisabled="selectedCourse == null"
+    @modal-closed="closeCurrentModal"
+    @left-button-clicked="backOrCancel"
+    @right-button-clicked="addItem"
+  >
+    <div class="newCourse-text">
+      {{ selecting ? 'Search Course Roster' : 'Selected Course' }}
+    </div>
+    <div v-if="selectedCourse != null" class="selected-course" data-cyId="newCourse-selectedCourse">
+      {{ selectedCourse.subject }} {{ selectedCourse.catalogNbr }}:
+      {{ selectedCourse.titleLong }}
+    </div>
+    <!-- show search bar if still selecting -->
+    <div v-if="selecting">
+      <course-selector
+        v-if="selectedCourse === null"
+        search-box-class-name="newCourse-dropdown"
+        :key="courseSelectorKey"
+        placeholder='"CS 1110", "Multivariable Calculus", etc'
+        :autoFocus="true"
+        @on-escape="closeCurrentModal"
+        @on-select="selectCourse"
+        data-cyId="newCourse-dropdown"
+      />
+      <div class="newCourse-text" v-if="selectedCourse === null">
+        <img
+          class="requirement-checker-warning-icon"
+          src="/src/assets/images/warning.svg"
+          alt="warning icon"
+          data-v-74b2c930=""
```

Comment:
What is this for?

Reply:
I think it's to make the warning icon the correct size, otherwise, it's much larger.

Reply:
Fixed with [c176926](https://github.com/cornell-dti/course-plan/pull/674/commits/c176926f95eef53a23ca4b6f3d5949776b58e0f6)